{"success": false, "error": "Traceback (most recent call last):\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/platform/solc.py\", line 544, in _run_solc\n    ret: Dict = json.loads(stdout)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/json/__init__.py\", line 357, in loads\n    return _default_decoder.decode(s)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/json/decoder.py\", line 337, in decode\n    obj, end = self.raw_decode(s, idx=_w(s, 0).end())\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/json/decoder.py\", line 355, in raw_decode\n    raise JSONDecodeError(\"Expecting value\", s, err.value) from None\njson.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/slither/__main__.py\", line 826, in main_impl\n    ) = process_all(filename, args, detector_classes, printer_classes)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/slither/__main__.py\", line 86, in process_all\n    compilations = compile_all(target, **vars(args))\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/crytic_compile.py\", line 637, in compile_all\n    compilations.append(CryticCompile(target, **kwargs))\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/crytic_compile.py\", line 117, in __init__\n    self._compile(**kwargs)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/crytic_compile.py\", line 548, in _compile\n    self._platform.compile(self, **kwargs)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/platform/solc.py\", line 154, in compile\n    targets_json = _get_targets_json(compilation_unit, self._target, **kwargs)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/platform/solc.py\", line 276, in _get_targets_json\n    return _run_solc(\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/platform/solc.py\", line 548, in _run_solc\n    raise InvalidCompilation(f\"Invalid solc compilation {stderr}\")\ncrytic_compile.platform.exceptions.InvalidCompilation: Invalid solc compilation Error: Source \"/Users/giacomoibba/Desktop/mainnet/node_modules/@openzeppelin/contracts/token/ERC20/ERC20.sol\" not found: File not found.\n --> /Users/giacomoibba/Desktop/mainnet/0a/0aA11C3dbc1C0855DA3eE00180A0F367fd54241C_BondlyTokenFoundersPool.sol:4:1:\n  |\n4 | import \"../node_modules/@openzeppelin/contracts/token/ERC20/ERC20.sol\";\n  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"/Users/giacomoibba/Desktop/mainnet/0a/BondlyTokenHolder.sol\" not found: File not found.\n  --> /Users/giacomoibba/Desktop/mainnet/0a/0aA11C3dbc1C0855DA3eE00180A0F367fd54241C_BondlyTokenFoundersPool.sol:18:1:\n   |\n18 | import \"./BondlyTokenHolder.sol\";\n   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"/Users/giacomoibba/Desktop/mainnet/node_modules/@openzeppelin/contracts/access/Ownable.sol\" not found: File not found.\n  --> /Users/giacomoibba/Desktop/mainnet/0a/0aA11C3dbc1C0855DA3eE00180A0F367fd54241C_BondlyTokenFoundersPool.sol:37:1:\n   |\n37 | import \"../node_modules/@openzeppelin/contracts/access/Ownable.sol\";\n   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"/Users/giacomoibba/Desktop/mainnet/node_modules/@openzeppelin/contracts/utils/ReentrancyGuard.sol\" not found: File not found.\n  --> /Users/giacomoibba/Desktop/mainnet/0a/0aA11C3dbc1C0855DA3eE00180A0F367fd54241C_BondlyTokenFoundersPool.sol:38:1:\n   |\n38 | import \"../node_modules/@openzeppelin/contracts/utils/ReentrancyGuard.sol\";\n   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"/Users/giacomoibba/Desktop/mainnet/0a/BondlyToken.sol\" not found: File not found.\n  --> /Users/giacomoibba/Desktop/mainnet/0a/0aA11C3dbc1C0855DA3eE00180A0F367fd54241C_BondlyTokenFoundersPool.sol:39:1:\n   |\n39 | import \"./BondlyToken.sol\";\n   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"/Users/giacomoibba/Desktop/mainnet/GSN/Context.sol\" not found: File not found.\n   --> /Users/giacomoibba/Desktop/mainnet/0a/0aA11C3dbc1C0855DA3eE00180A0F367fd54241C_BondlyTokenFoundersPool.sol:123:1:\n    |\n123 | import \"../GSN/Context.sol\";\n    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"/Users/giacomoibba/Desktop/GSN/Context.sol\" not found: File not found.\n   --> /Users/giacomoibba/Desktop/mainnet/0a/0aA11C3dbc1C0855DA3eE00180A0F367fd54241C_BondlyTokenFoundersPool.sol:352:1:\n    |\n352 | import \"../../GSN/Context.sol\";\n    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"/Users/giacomoibba/Desktop/mainnet/0a/IERC20.sol\" not found: File not found.\n   --> /Users/giacomoibba/Desktop/mainnet/0a/0aA11C3dbc1C0855DA3eE00180A0F367fd54241C_BondlyTokenFoundersPool.sol:353:1:\n    |\n353 | import \"./IERC20.sol\";\n    | ^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"/Users/giacomoibba/Desktop/math/SafeMath.sol\" not found: File not found.\n   --> /Users/giacomoibba/Desktop/mainnet/0a/0aA11C3dbc1C0855DA3eE00180A0F367fd54241C_BondlyTokenFoundersPool.sol:354:1:\n    |\n354 | import \"../../math/SafeMath.sol\";\n    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\n\n", "results": {}}
